packages/repository/src/__tests__/fixtures/mixins/categoryPropertyMixin.ts

```diff
@@ -0,0 +1,26 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
```

Comment:
We use kebab-case in file names. Please rename this file to `category-property-mixin.ts`.